core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -143,14 +144,29 @@ class SparkHadoopUtil extends Logging {
    * Returns a function that can be called to find Hadoop FileSystem bytes read. If
    * getFSBytesReadOnThreadCallback is called from thread r at time t, the returned callback will
    * return the bytes read on r since t.
-   *
-   * @return None if the required method can't be found.
```

Comment:
this doesn't return a `None`, but the doc is still corrected about the behavior.